bc_obps/common_utils/tests/test_email_service.py

```diff
@@ -0,0 +1,137 @@
+import pytest
+from datetime import datetime
+from uuid import UUID
+from unittest.mock import patch
+from common_utils.email.email_service import EmailService
+
+
+@pytest.fixture
+def email_service():
+    return EmailService()
+
+
+# NOTE: Most of this mock data was pulled from sample payloads from the CHES API docs
+# at https://ches.api.gov.bc.ca/api/v1/docs
+
+
+@pytest.fixture
+def email_data():
+    return {
+        'bodyType': 'text',
+        'body': 'This is the body of a test email for BCIERS Email Service',
+        'from': 'ggircs@gov.bc.ca',
+        'subject': 'Automated Test of Email_Service',
+        'to': ['baz@gov.bc.ca'],
+    }
+
+
+@pytest.fixture
+def email_template():
+    return {
+        'attachments': [
+            {
+                'content': 'PGI+SGVsbG8gV29ybGRcITwvYj4=',
+                'contentType': 'string',
+                'encoding': 'base64',
+                'filename': 'testfile.txt',
+            }
+        ],
+        'bodyType': 'html',
+        'body': '{{ something.greeting }} {{ something.target }} content',
+        'contexts': [
+            {
+                'bcc': ['foo@gov.bc.ca'],
+                'cc': ['fizz@gov.bc.ca'],
+                'context': {'something': {'greeting': 'Hello', 'target': 'World'}, 'someone': 'user'},
+                'delayTS': 1570000000,
+                'tag': 'tag',
+                'to': ['baz@gov.bc.ca'],
+            }
+        ],
+        'encoding': 'utf-8',
+        'from': 'example@gov.bc.ca',
+        'priority': 'normal',
+        'subject': 'Hello {{ someone }}',
+    }
+
+
+def test_get_token(email_service):
+    email_service._get_token()
```

Comment:
Does this unit test hit the CHES API? we might want to mock the `requests` package methods instead and assert it's been called with the right parameters

Reply:
Yes, right now both the get_token and health_check tests actually touch CHES API. Good call - they should be replaced with mock requests instead.

Reply:
Refactored tests so that they don't actually use the CHES API (except for the skipped test that sends a real email when un-skipped)